elmslie-plugin/src/main/resources/templates/PresentationActor.kt.ftl

```diff
@@ -0,0 +1,22 @@
+package ${presentation_package_name}
+
+import vivid.money.elmslie.core.store.Actor
+import ${presentation_package_name}.${domain_name}Command.*
+<#if split_events>
+import ${presentation_package_name}.${domain_name}Event.Internal
+<#else>
+import ${presentation_package_name}.${domain_name}Event.*
+</#if>
+import io.reactivex.rxjava3.core.Observable
+import javax.inject.Inject
+
+internal class ${domain_name}Actor @Inject constructor(
+    // your dependencies
+) : Actor<${domain_name}Command, ${domain_name}Event> {
```

Comment:
I guess here it should be `Internal` as well, i couldn't compile the sample without it

Reply:
Fixed